frontend/apps/filemanager/filemanagerutil.lua

```diff
@@ -138,34 +130,64 @@ end
 
 -- Generate "Reset" file dialog button
 function filemanagerutil.genResetSettingsButton(file, caller_callback, button_disabled)
+    file = ffiutil.realpath(file)
+    local sidecar_file = DocSettings:hasSidecarFile(file)
+    local custom_cover_file = DocSettings:findCoverFile(file)
+    local custom_metadata_file = DocSettings:getCustomMetadataFile(file)
```

Comment:
Just wondering: any reason for these 2 methods (somehow similar in purpose) are named differently?
Why not `findCoverFile` and `findCustomMedatadaFile` - or `getCoverFile` and `getCustomMedataFile` ?

Reply:
We use
https://github.com/koreader/koreader/blob/ed2ea6803f4b9be6b065f43590229dea4fe51426/frontend/docsettings.lua#L429
when we have opened doc_settings (to use the cache).